crates/uv/src/commands/tool/common.rs

```diff
@@ -123,47 +123,52 @@ pub(crate) fn install_executables(
         return Ok(ExitStatus::Failure);
     }
 
-    // Check if they exist, before installing
-    let mut existing_entry_points = target_entry_points
-        .iter()
-        .filter(|(_, _, target_path)| target_path.exists())
-        .peekable();
```

Comment:
@zanieb -- I noticed this logic is such that we delete the existing entrypoints, then we write them in the `for (name, source_path, target_path) in &target_entry_points ` loop below. For self-replace purposes, it's easier to model it as a replace rather than a delete + write. Do you think that's a problem?

Reply:
I think that's fine, though it'll change the error behavior. Perhaps we should defer errors until the end so we accomplish as many replacements as we can?